src/pydata_sphinx_theme/assets/styles/sections/_header.scss

```diff
@@ -89,21 +89,49 @@
       align-items: baseline;
     }
 
-    li.pst-header-nav-item {
```

Comment:
I realized that with #1846, I no longer need a special class to select the navbar text links. Now they can be selected with: `.navbar-nav > .nav-item`.

TODO: check if this line needs a `>`